### ISSUE TITLE
Some XML entities are not being resolved

### ISSUE THREAD

**Issue by jabrah** [jhu-digital-manuscripts/rosa2#296]
See Monmouth 17v
![image](https://user-images.githubusercontent.com/5167587/34178951-03217a98-e4d7-11e7-98c5-c9777b98d03b.png)

Text is not escaped in this way in the transcriptions:
![image](https://user-images.githubusercontent.com/5167587/34179025-4e487710-e4d7-11e7-8a20-e5ca4ef247f4.png)

**Comment by jabrah** [comment]
Possibly related: #283

**Comment by jabrah** [comment]
``` java
writer.writeCharacters(StringEscapeUtils.escapeHtml4(transcription.toString()));
```
While translating AOR transcriptions to HTML, the HTML is output from an XML writer. In lines like this where we directly write AOR data, we do HTML escaping to prevent abuse. This has a side effect of also escaping special characters, such as accents or Greek characters.

`á` -> `&aacute;`

This escaped sequence is then written as XML characters, presumably in a text node. This will further escape some characters.

`&aacute;` -> `&amp;aacute;`

This final string will render as seen in the screenshots above: `&aacute;`

**Comment by jabrah** [comment]
Resolved by : b35e05c1f44d4073157e85e8b6c2bda4986356aa
